lib/server/db/postgres/postgres_test.go

```diff
@@ -133,6 +132,10 @@ var _ = Describe("Postgres", func() {
 			_, err := db.CreateDatabase()
 			Expect(err).To(HaveOccurred())
 			Expect(err.Error()).Should(ContainSubstring("Failed to create Postgres database: Failed to execute create database query: error creating database"))
+
+			exists, err := exists(db.SqlxDB, db.dbName)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(exists).To(BeFalse())
```

Comment:
The existence query should be a separate context and assertion. Something like `When("the database already exists") ... It("does not attempt to create the database")...`.